dune/grid/cpgrid/GridHelpers.hpp

```diff
@@ -399,6 +399,74 @@ beginCellCentroids(const Dune::CpGrid& grid);
 double cellCentroidCoordinate(const Dune::CpGrid& grid, int cell_index,
                               int coordinate);
 
+/// \brief Get the centroid of a cell.
+/// \param grid The grid whose cell centroid we query.
+/// \param cell_index The index of the corresponding cell.
+const double* cellCentroid(const Dune::CpGrid& grid, int cell_index);
+
+/// \brief Get the volume of a cell.
+/// \param grid The grid the cell belongs to.
+/// \param cell_index The index of the cell.
+double cellVolume(const  Dune::CpGrid& grid, int cell_index);
+
+/// \brief An iterator over the cell volumes.
+class CellVolumeIterator
+    : public Dune::RandomAccessIteratorFacade<CellVolumeIterator, double, double, int>
```

Comment:
Thanks for noticing. Just moved the include fof iteratorfacades.hh from OPM/dune-cornerpoint#323 to here